client/src/components/Todo/Todo.jsx

```diff
@@ -0,0 +1,48 @@
+import PropTypes from "prop-types";
+
+function Todo(props) {
```

Comment:
This is a good example of a view component that doesn't contain any blocking / unnecessary logic